tests/zepkit/test/package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "tests-kit-test",
+  "private": true,
+  "version": "2.3.0",
+  "description": "Integration tests for ZepKit and an unpack command",
+  "author": "santiago@zeppelin.solutions",
```

```suggestion
  "author": "igor@zeppelin.solutions",
```
He who kickstarts the project gets to author it :-)